mapk_demon/transmitters.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import division
+
+from functools import partial
+
+from pyro import sample
+from pyro.distributions import LogNormal
+from torch import tensor
+
+
+def g1(a):
+    return a / (a + 1)
+
+
+def g2(a):
+    return a**2 / (a**2 + a + 1)
+
+
+def f(mu, N):
+    return N ** (0.5) * mu + mu
+
+
+constants = {
+    "t_3k": 1.2,
+    "t_2k": 1.2,
+    "t_k": 0.003
+}
+
+params = {
+    "alpha_3k": 500,
+    "alpha_2k": 500,
+    "alpha_k": 500,
+    "nu_3k": .15,
+    "nu_2k": .15,
+    "nu_k": .15
+}
+
+
+def f_map3k(E1, N_3k):
+    total_3k = constants['t_3k']
+    alpha_3k = params['alpha_3k']
+    nu_3k = params['nu_3k']
+
+    map3k_mu = total_3k * g1(E1 * (alpha_3k/nu_3k))
+
+    return f(map3k_mu, N_3k)
+
+
+def f_map2k(map3k, N_2k):
+    total_2k = constants['t_2k']
+    alpha_2k = params['alpha_2k']
+    nu_2k = params['nu_2k']
+
+    map2k_mu = total_2k * g2(map3k * (alpha_2k / nu_2k))
+
+    return f(map2k_mu, N_2k)
+
+
+def f_mapk(map2k, N_k):
+    total_k = constants['t_k']
+    alpha_k = params['alpha_k']
+    nu_k = params['nu_k']
+
+    mapk_mu = total_k * g2(map2k * (alpha_k / nu_k))
+
+    return f(mapk_mu, N_k)
+
+
+def mapk_signaling(noise_dists):
+
+    E1 = 1.
```

Comment:
You will want to sample E1 from some distribution, so that you can fix it to certain values with `condition` or `do`.  The default value is 3e-5 in the model.  One possibility is a uniform from 1.5e-5 to 6e-5.  Or a Gamma.